copy/copy.go

```diff
@@ -361,6 +365,44 @@ func supportsMultipleImages(dest types.ImageDestination) bool {
 	return false
 }
 
+// compareImageDestinationManifestEqual compares the `src` and `dest` image's manifests, returning the 'true' if they compare equal.
+// The destination manifest is always fetched.
+func compareImageDestinationManifestEqual(ctx context.Context, options *Options, src types.Image, dest types.ImageDestination) (bool, []byte, string, digest.Digest, error) {
+
+	srcManifest, _, err := src.Manifest(ctx)
+	if err != nil {
+		return false, nil, "", "", errors.Wrapf(err, "Error reading manifest from image")
+	}
+
+	srcManifestDigest, err := manifest.Digest(srcManifest)
+	if err != nil {
+		return false, nil, "", "", errors.Wrapf(err, "Error calculating manifest digest")
+	}
+
+	destImageSource, err := dest.Reference().NewImageSource(ctx, options.DestinationCtx)
+	if err != nil {
+		return false, nil, "", "", nil
+	}
+
+	destManifest, destManifestType, err := destImageSource.GetManifest(ctx, nil)
+	if err != nil {
+		return false, nil, "", "", nil
+	}
+
+	destManifestDigest, err := manifest.Digest(destManifest)
+	if err != nil {
+		return false, nil, "", "", errors.Wrapf(err, "Error calculating manifest digest")
+	}
+
+	logrus.Debugf("Comparing source and destintion manifest digests: srcManifestDigest=%v, destManifestDigest=%v", srcManifestDigest, destManifestDigest)
```

Comment:
```suggestion
	logrus.Debugf("Comparing source and destination manifest digests: srcManifestDigest=%v, destManifestDigest=%v", srcManifestDigest, destManifestDigest)
```
or maybe just something like
```suggestion
	logrus.Debugf("Source and destination manifest digests: %v vs. %v", srcManifestDigest, destManifestDigest)
```
to be less verbose in a many-image `skopeo sync` log.